### PR TITLE
Add Hubspot authorise URL

### PR DESCRIPTION
**Pull request by sophiemachin**

### CODE REVIEW

File: thiscovery_admin/templates/hubspot.html [hunk 1]
```diff
@@ -14,6 +14,7 @@ <h2>
     </h2>
     <p>
         <a href="https://app.hubspot.com/oauth/authorize?scope=contacts%20timeline&redirect_uri=https://www.hubspot.com/auth-callback&client_id={{ client_id }}" target="_blank">Install</a>
+        <a href="https://app.hubspot.com/oauth/authorize?client_id={{ client_id }}&redirect_uri=https://abcd-123-45-88-14.ngrok.io/hubspot&scope=contacts%20timeline" target="_blank">Authorise</a>
```
Comment (afs25) [inline thiscovery_admin/templates/hubspot.html:17]: I don't like that the variable bit abcd-123-45-88-14 in this URL is hardcoded. Can we implement a better solution? I know that thiscovery_admin doesn't have a dev_config file, but getting the settings file in use to export to os.env and then reading the value would work. Let's discuss this at the next dev meeting.

**Review by afs25 — CHANGES_REQUESTED** [review]

File: thiscovery_admin/templates/hubspot.html (hunk 1 above)
Comment (sophiemachin) [reply thiscovery_admin/templates/hubspot.html:17]: Apologies I've only just seen this message. I'll have a go at your suggestion.

Comment (sophiemachin) [reply thiscovery_admin/templates/hubspot.html]: @afs25  I've made that change

File: thiscovery_admin/projects/views.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 def index(request):
     # return HttpResponse('Authenticate HubSpot API')
     hubspot_connection = get_secret("hubspot-connection")
     api_key = hubspot_connection["api-key"]
     client_id = hubspot_connection["client-id"]
-    context_dict = {"client_id": client_id, "namespace": get_aws_namespace()}
+    context_dict = {
+        "client_id": client_id,
+        "namespace": get_aws_namespace(),
+        "ngrok_address": os.environ.get("NGROK_ADDRESS", ""),
```
[the remark this answers is not in the document]
Comment (sophiemachin) [reply thiscovery_admin/projects/views.py]: I've added a message

**Review by afs25 — APPROVED** [review]

**Pull request merged**